07.md

```diff
@@ -13,6 +13,9 @@ That object must define the following methods:
 ```
 async window.nostr.getPublicKey(): string // returns a public key as hex
 async window.nostr.signEvent(event: Event): Event // takes an event object and returns it with the `sig`
+async window.nostr.getRelays(): { [url: string]: RelayPolicy } // returns a map of relays
```

Comment:
RelayPolicy is not defined. Maybe this is clearer?

```suggestion
async window.nostr.getRelays(): { [url: string]: {read: boolean, write: boolean} } // returns a map of relay urls to relayPolicies
```

Reply:
good idea. thanks! done.